nsxt/resource_nsxt_vpc_group.go

```diff
@@ -0,0 +1,36 @@
+/* Copyright © 2024 Broadcom, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+func resourceNsxtVPCGroup() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceNsxtVPCGroupCreate,
+		Read:   resourceNsxtVPCGroupRead,
+		Update: resourceNsxtVPCGroupUpdate,
+		Delete: resourceNsxtVPCGroupDelete,
+		Importer: &schema.ResourceImporter{
+			State: nsxtPolicyPathResourceImporter,
```

Comment:
This comment is not specific to group, but rather to all vpc resources - it seems to me that we need VPC-specific Importer that would verify that the path is VPC path

Reply:
You mean, call `nsxtPolicyPathResourceImporterHelper()` and fail if no `project_id` or `vpc_id` are specified? Yeah that could be helpful.

Reply:
yes, a  wrapper for `nsxtPolicyPathResourceImporter` that also verifies we have a VPC path. I don't think we should validate the object ids against what we might have and not have in config.
maybe its a good idea to create a generic importer that verifies specified path structure, and vpc would be just a use case for that.